Tests/feaLib/data/STAT_test.ttx

```diff
@@ -8,9 +8,15 @@
     <namerecord nameID="256" platformID="3" platEncID="1" langID="0x409">
       Roman
     </namerecord>
+    <namerecord nameID="256" platformID="1" platEncID="0" langID="0x0" unicode="True">
+      Roman
```

Comment:
these duplicated entries are an unintentional byproduct of using addMultilingualName early on (which has an option mac=True), when given a list of elidedFallbackName records. We only want to add the records that we are requested, we don't want to automatically add "mac" names unless explicitly requested. Using the simple addName (as I commented above) should do what we want and remove these duplicate entries.